plugins/caddy/httpcache.go

```diff
@@ -216,6 +216,9 @@ func (s *SouinCaddyMiddleware) FromApp(app *SouinApp) error {
 	if dc.CacheName == "" {
 		s.Configuration.DefaultCache.CacheName = appDc.CacheName
 	}
+	if !dc.DisableCoalescing && appDc.DisableCoalescing {
+		s.Configuration.DefaultCache.DisableCoalescing = appDc.DisableCoalescing
+	}
```

Comment:
IMHO that's too sensitive to override the `disable_coalescing` from the app because we can't detect if the user omitted the `disable_coalescing` directive in the handler or if it's set as `false`.